.github/pull_request_template.md

```diff
@@ -21,6 +21,9 @@ As the author of this PR, please check off the items in this checklist:
 - [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
 - [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
   functionality, content, code)
+- [ ] User-facing changes have a [Tekton Enhancement Proposal](https://github.com/tektoncd/community/tree/main/teps) in an
+  [*implementable*](https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-workflow)
+  state that is referenced in the [changes](#changes) section.
```

Comment:
How about: 

are the changes part of any TEP? If so, is the TEP approved and in `implementable` state?

Reply:
this is simpler, but excludes about the part of what needs to be in a TEP